keras/applications/efficientnet.py

```diff
@@ -322,8 +327,9 @@ def round_repeats(repeats):
 
   # Build stem
   x = img_input
-  x = layers.Rescaling(1. / 255.)(x)
-  x = layers.Normalization(axis=bn_axis)(x)
+  if include_preprocessing:
+    x = layers.Rescaling(1. / 255.)(x)
+    x = layers.Normalization(axis=bn_axis)(x)
```

Comment:
The normalization layer might contain weights, so this change might break existing h5 weights. Could u make sure the unit tests also passes when include_preprocessing is False?

Reply:
Yes @qlzh727 the test breaks and it appears that it has weights. So instead of skipping the rescaling and normalization can we do something like 
`x= tf.math.multiply(tf.math.add(x, tf.math.subtract(0. tf.math.minimum(x))), tf.math.divide_no_nan(255.0, tf.math.maximum(x)))` to first rescale input to range [0, 255] and then run the model as is, if `include_preprocessing=False` ?